### PR TITLE
Hogsters

### PR DESCRIPTION
**Pull request by miguelhunter**
Apologies for the double requests. Please disregard the first one and use the 'updated' one.  Thanks!

### CODE REVIEW

File: Hogsters [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "tags": [
+        "Hogsters"
+    ],
+    "policies": [
+        "1ad2d49637ae94776fa8ec98d05f23e00a1e03b2caefa45d79dd364f",
+        "444b25fa8f39906ca302639d66005f6e66756f677b41e4e546cba33a",
+        "2bf7e107f0a500f9f51d5d577cc3a78c9b5f813bf4f6fb39eff92d20",
+        "c9ff8f4bd2edc7b8f11feaa6b7c5087a61a909359c201ae1191b8820",
+        "ce3aa279f28f41802be5c9208be93352e7f599f215bffbd770b0c59c",
+        "7dc5a4ba313d62430ac0bc6c192b234d60089a76549e49b40b536e8a",
```
Comment (itsmestale) [inline Hogsters:12]: remove comma from this line

**Review by itsmestale — CHANGES_REQUESTED** [review]
remove comma from last policy id

**Comment by itsmestale** [comment]
also please read the readme regarding twitter verification

File: Hogsters (hunk 1 above)
Comment (miguelhunter) [reply Hogsters:12]: done!

**Review by Dean00K — APPROVED** [review]

**Pull request merged**